pkg/kafsar/group_coordinator_standalone_test.go

```diff
@@ -26,7 +26,7 @@ import (
 var (
 	clientId         = "consumer-a1e12365-ddfa-43fc-826e-9661fb54c274-1"
 	groupId          = "test-group-id"
-	memberId         = "test-member-id"
+	memberId         = ""
```

Comment:
I suggest create two test instead of change value

Reply:
it should be empty, when join group fist time